airbyte-integrations/connectors/source-amplitude/acceptance-test-config.yml

```diff
@@ -27,6 +27,8 @@ acceptance_tests:
             bypass_reason: "This stream is empty due to free subscription plan for the sandbox."
           - name: "annotations"
             bypass_reason: "This stream is empty due to free subscription plan for the sandbox."
+          - name: "average_session_length"
```

Comment:
Should we add the `integration tests` for this stream?

Reply:
There are no such tests at all currently, so will create a ticket to add them